inlong-manager/manager-docker/Dockerfile

```diff
@@ -37,10 +37,14 @@ ENV AUDIT_PROXY_URL=127.0.0.1:10081
 # JVM env
 ENV MANAGER_JVM_HEAP_OPTS="-XX:+UseContainerSupport -XX:InitialRAMPercentage=40.0 -XX:MaxRAMPercentage=80.0 -XX:-UseAdaptiveSizePolicy"
 WORKDIR /opt/inlong-manager
-
-# add tarball from manager output
-ARG MANAGER_TARBALL
```

Comment:
docker/build-docker-images.sh has used MANAGER_TARBALL at line 122, need to change into version

Reply:
fix it.